code/game/objects/structures/signs/_signs.dm

```diff
@@ -30,111 +34,118 @@
 		return
 	user.examinate(src)
 
+GLOBAL_VAR(editable_sign_types)
+/proc/populate_editable_sign_types() //The first time a pen is used on any sign, this populates the above, a global list of all the signs that you can set a sign backing to with a pen.
```

Comment:
code doc this

Reply:
also this should probably be done in a NO_FIRE subsystem

Reply:
For now I documented it, and will figure out the subsystem thing later. Thanks.

Reply:
Opted to just move it to _globalvars and make it GLOBAL_LIST_EMPTY